src/hmem_cuda.c

```diff
@@ -760,6 +760,19 @@ bool cuda_is_gdrcopy_enabled(void)
 	return hmem_cuda_use_gdrcopy;
 }
 
+#if CUDA_VERSION >= 11030
```

Comment:
Just noticed this: your earlier change used `CUDART_VERSION`; this one uses `CUDA_VERSION`. What's the distinction?

Reply:
I forgot I did that... RT == Runtime API.  If a function call starts with cuda, it uses the runtime API, if it starts with cu, it used the driver API.... Should be RT.

Reply:
done